### DISCUSSION TITLE
how do i set this up on Windows (visual studio?)

### DISCUSSION THREAD

**Discussion by wttldt** [ggml-org/whisper.cpp#491] [Q&A]
How do I set this up on Windows: https://github.com/ggerganov/whisper.cpp

The instructions are unclear to me. I forked the repository into my own, put the windows yaml file into my .github\workflows folder now I'm not sure where to go.

**Comment by RndyP** [comment]
Are you incorporating whisper into an app, or just trying to get main, command or stream up and running as a console app? If it's the former, you just add ggml.* and Whisper.* to your project and you are good to go. If it's the latter, create a new console app with the Wizard, and simply replace main generated by the Wizard with the main from whisper here.

**Reply by wttldt** [reply]
i just want it on my desktop. i can run whisper from my CMD my typing "whisper audio.mp3" then it goes just fine. is there a way I can get it to work in visual studio? i'm very new to this, could you please give me step by step instructions on how to get this done? thank you!

**Reply by Mike-Bell** [reply]
I'm confused what you want. What do you mean "i just want it on my desktop"? I'm sure you're aware that you can move the exe wherever you'd like, so I'm not sure what you're looking for.

**Reply by wttldt** [reply]
I just want to run it in visual studio for Windows 10, to stream audio and transcribe it to visual studio (with no timestamps). Is that possible?

**Reply by albino1** [reply]
Maybe just use this Windows GUI version of whisper.cpp instead of dealing with Visual Studio:

https://github.com/Const-me/Whisper/

**Reply by wttldt** [reply]
thanks for that suggestion, this is great
